### PR TITLE
[T4A4][W13-A4]Lee Yan Hwa

### PR DESCRIPTION
**Pull request by leeyh20**
Ready for Review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -56,4 +56,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+       return "Address: " + value;
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Address.java:62]: Take note of the magic strings used

File: src/seedu/addressbook/data/person/Name.java [hunk 2]
```diff
@@ -1,15 +1,15 @@
 package seedu.addressbook.data.person;
 
-import seedu.addressbook.data.exception.IllegalValueException;
-
 import java.util.Arrays;
 import java.util.List;
 
+import seedu.addressbook.data.exception.IllegalValueException;
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Name.java:6]: Unused imports should be removed.

File: src/seedu/addressbook/data/person/Printable.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Printable.java:3]: Remember to include the header comments for new classes

**Review by K1ang — COMMENTED** [review]
@leeyh20 Some comments added. Please close the PR after reading comments. 
Remember to update the tests and documentations too!

**Comment by leeyh20** [comment]
thanks!

**Pull request closed without merging**